src/main/java/chess/manager/ChessManager.java

```diff
@@ -41,26 +34,26 @@ private void validateTurn(final Position source) {
         }
     }
 
-    private boolean isTargetKing(Position target) {
-        return board.isTargetKing(target);
+    public void changeTurn() {
+        turn = turn.reverse();
     }
 
-    public Path getReachablePositions(final ShowCommand command) {
-        validateSourcePiece(command.source());
-        return board.getAbleToMove(command.source());
+    public boolean isDiedKing() {
```

Comment:
```suggestion
    public boolean isKingDead() {
```

의도에 맞는 더 적절한 이름일 것 같네요 😉